docker/Dockerfile.fedora

```diff
@@ -40,3 +24,12 @@ RUN mkdir -p build && \
 FROM fedora:latest
 COPY --from=builder /opt/libabigail /opt/libabigail
 ENV PATH=/opt/libabigail/bin:$PATH
+
+# Runtime dependencies
+RUN dnf install -y \
+    libxml2-devel \
```

Comment:
libxml2-devel shouldn't be needed in the runtime.

Reply:
However you will likely need elfutils-libelf elfutils-libs and libstdc++

Reply:
Removed. https://github.com/woodard/libabigail/pull/10/commits/d289eabd0b6b9de21673a5b1d6d1e69c4745080f